src/main/java/de/komoot/photon/App.java

```diff
@@ -134,9 +135,17 @@ private static void startNominatimImport(CommandLineArgs args, Server esServer,
      * @param esNodeClient
      */
     private static void startApi(CommandLineArgs args, Client esNodeClient) {
-        setPort(args.getListenPort());
-        setIpAddress(args.getListenIp());
-
+        port(args.getListenPort());
+        ipAddress(args.getListenIp());
```

Comment:
This looks unrelated. Any particular reason to change it?

Reply:
The old versions are deprecated in spark but functionally exactly the same, changed here to get rid of the warnings.